test/distributed/test_c10d.py

```diff
@@ -3523,6 +3551,12 @@ def test_builtin_ddp_comm_hooks_nccl(self):
     def test_ddp_comm_hook_allreduce_hook_nccl_grad_is_view(self):
         self._test_ddp_comm_hook_allreduce_hook_nccl(gradient_as_bucket_view=True)
 
+    @requires_nccl()
+    @skip_if_lt_x_gpu(2)
+    @skip_if_rocm
+    def test_default_ddp_comm_hooks_nccl_is_view(self):
+        self._test_default_ddp_comm_hooks_nccl(gradient_as_bucket_view=True)
```

Comment:
Thanks for adding the test with `gradient_as_bucket_view`! LGTM once all tests pass